firestore/integration_test.go

```diff
@@ -705,6 +705,90 @@ func TestIntegration_Query(t *testing.T) {
 	}
 }
 
+func TestIntegration_QueryDocuments_LimitToLast_Fail(t *testing.T) {
+	ctx := context.Background()
+	coll := integrationColl(t)
+	q := coll.Select("q").OrderBy("q", Asc).LimitToLast(1)
+	gotDocs, err := q.Documents(ctx).GetAll()
+	if err == nil {
+		t.Errorf("got %v docs, want error", len(gotDocs))
+	}
+}
+
+func TestIntegration_QueryGet(t *testing.T) {
+	ctx := context.Background()
+	coll := integrationColl(t)
+	h := testHelper{t}
+	var wants []map[string]interface{}
+	for i := 0; i < 3; i++ {
+		doc := coll.NewDoc()
+		// To support running this test in parallel with the others, use a field name
+		// that we don't use anywhere else.
+		h.mustCreate(doc, map[string]interface{}{"q": i, "x": 1})
+		wants = append(wants, map[string]interface{}{"q": int64(i)})
+	}
+	q := coll.Select("q").OrderBy("q", Asc)
+	for i, test := range []struct {
+		q    Query
+		want []map[string]interface{}
+	}{
+		{q, wants},
+		{q.Where("q", ">", 1), wants[2:]},
+		{q.WherePath([]string{"q"}, ">", 1), wants[2:]},
+		{q.Offset(1).Limit(1), wants[1:2]},
+		{q.StartAt(1), wants[1:]},
+		{q.StartAfter(1), wants[2:]},
+		{q.EndAt(1), wants[:2]},
+		{q.EndBefore(1), wants[:1]},
+		{q.LimitToLast(2), wants[1:]},
```

Comment:
Also it needs to add `Limit()` here, I think

Reply:
There is a `limit()` call few lines upper